setup.py

```diff
@@ -0,0 +1,7 @@
+from setuptools import setup
+
+setup(
+	name='sibispy',
+	version='1.0',
+	py_modules=["sibispy"]
+)
```

Comment:
I'd like to see the various entrypoints and scripts from the `cmd` directory get added to this.
Also missing dependencies.
see `https://github.com/sibis-platform/hivalc-data-integration/blob/master/setup.py` as an example.
Also if we can get the tests declared in the setup, that would be good.  We may need to discuss this a bit as the sibispy requires quite a bit of configuration to work, namely a `~/.sibis-general-config.yml` and a mapping to an `operations` directory that stores `sibis-test-config.yml` and `sibis-sys-config.yml` and `special-cases.yml`.

Reply:
I added the entry points. We can work on the setup and get everything fixed.